src/modules/Game.class.js

```diff
@@ -23,22 +23,156 @@ class Game {
   constructor(initialState) {
     // eslint-disable-next-line no-console
     console.log(initialState);
+
+    // Ініціалізуємо гральну дошку
+    this.defaultState = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ];
+
+    this.board = initialState || JSON.parse(JSON.stringify(this.defaultState));
+    this.score = 0; // Початкови рахунок
+    this.status = 'idle'; // Початковий статус гри
+  }
+
+  moveLeft() {
```

Comment:
it works even before I click "start" button - it should be fixed:

![image](https://github.com/user-attachments/assets/e07c71bb-8a3f-4ce7-a7cc-1e44cd0e7d0f)